docs/eden/treaty/response.md

```diff
@@ -15,17 +15,17 @@ head:
 ---
 
 # Response
-Once fetch method is called, Eden Treaty return an Promise with object as follows:
+Once the fetch method is called, Eden Treaty return a promise containing an object with the following properties:
 - data - returned value of the response (2xx)
 - error - returned value from the response (>= 3xx)
 - response `Response` - Web Standard Response class
 - status `number` - HTTP status code
-- headers `FetchRequestInit['headers']` - response's headers
+- headers `FetchRequestInit['headers']` - response headers
 
-Once returned, you must provide an error handling to ensure that value is truly returned to access the value, otherwise the value will be nullable.
+Once returned, you must provide error handling to ensure that the response data value is unwrapped, otherwise the value will be nullable. Elysia provides a `error()` helper function to handle the error, and Eden will provide type narrowing for the error value. 
 
 ```typescript twoslash
-import { Elysia, t } from 'elysia'
+import { Elysia, error, t } from 'elysia'
```

Comment:
Please undo this. The `error` function from handler-context is used:

```ts
app.post('/user', ({ body: { name }, error }) => { })
//                                     ^
```

feel free to file an issue [here](https://github.com/elysiajs/elysia/issues) if you encounter any problems with the function